Countries.py

```diff
@@ -162,6 +162,14 @@ def flush_queue(self):
             for event in pygame.event.get():
                 flushing = True
 
+    def proximity(self, up, down):

```

Comment:
Seem complicated for what it does.  It is a distance calculation using each axis.  x_max is the same as x_min, and y_max is the same as y_min, could instead use down.pos[0] and down.pos[1], or calculate absolute value of difference in each coordinate.  look at pygame's collision detection functions.